matRad_calcSigmaIni.m

```diff
@@ -0,0 +1,73 @@
+function [sigmaIni] = matRad_calcSigmaIni(baseData,rays,SSD)
```

Comment:
1. In its current implementation this function is not correct because different rays can have different SSD; you only compute for the same SSD.
2. I doubt that this function provides a significant speedup. How much relative time is spent in this function anyways? You really make things way more complicated which heavily decreases readability; we should consider simply going back to the original implementation...

Let's discuss!